pkgs/tools/text/epubcheck/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchzip
+, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "epubcheck";
+  version = "4.2.0";
+
+  src = fetchzip {
```

Comment:
fetchFromGitHub is preferred here

Reply:
The source files aren't compiled and it doesn't seem like it's easy to create java based derivations.

Reply:
Ah yeah that makes sense.